pkg/network/dhcp/configurator.go

```diff
@@ -9,15 +9,15 @@ import (
 	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
 )
 
-const dhcpStartedDirectory = "/var/run/kubevirt-private"
+const defaultDHCPStartedDirectory = "/var/run/kubevirt-private"
```

Comment:
I would change the commit message of this commit. We are not mocking the `dhcpConfigurator` because of the described failure. We mock is from the same reason `podInfraConfigurator` is mocked, we want to test only `podNic` functionality.

EDIT: now I see the commit is actually not mocking the `dhcpConfigurator`. Why?

Reply:
At last commit we stub the DHCP locking file to ensure DHCP server is up so I suppose this is coverthing this.